alignak/objects/config.py

```diff
@@ -1103,13 +1103,14 @@ def read_config_buf(self, buf):  # pylint: disable=R0912
 
             line_nb += 1
             # Remove comments
-            line = split_semicolon(line)[0].strip()
```

Comment:
I agree with @Seb-Solon ... we should not remove this line because it is used to remove comments starting with a semi-colon on the same line.